manager.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash -eu
+#
+# Copyright 2018 The Outline Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IFS="
+"
+
+declare -a FILES=(
+  # macOS.
+  Outline-Manager.dmg
+  Outline-Manager.dmg.blockmap
+  latest-mac.yml
+  Outline-Manager.zip
+  # Windows.
+  Outline-Manager.exe
+  Outline-Manager.exe.blockmap
+  latest.yml
+  # Linux.
+  Outline-Manager.AppImage
+  latest-linux.yml
+)
+
+function usage() {
+  cat <<-EOM
+Usage: $0 <tag>
```

Comment:
Could you add a short explanation of what this script does? A more descriptive filename could also be useful - i.e. release_manager.sh

Reply:
Both done!